http-server/src/main/java/io/airlift/http/server/HttpServerInfo.java

```diff
@@ -56,10 +55,10 @@ public HttpServerInfo(HttpServerConfig config, NodeInfo nodeInfo)
 
         if (config.isAdminEnabled()) {
             if (config.isHttpsEnabled()) {
-                adminUri = buildUri("https", InetAddresses.toUriString(nodeInfo.getInternalIp()), config.getAdminPort());
```

Comment:
toUriString wraps IPv6 addresses with square brackets. This PR makes getInternalAddress return a String that's constructed from `InetAddresses.toAddrString(ip)`, so it's not equivalent.